atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -677,10 +685,10 @@ def run(self):
             self.log.info("Not promoting failed build to koji")
             return
 
-        koji_metadata, output_files = self.get_metadata()
+        session = get_koji_session(self.workflow, self.koji_fallback)
```

Comment:
It might be worth to assign `self.koji_session` so it doesn't have to be passed around.